src/auth_client_demo_assets/svelte/src/routes/+page.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+  import LoggedIn from "../components/LoggedIn.svelte";
+  import LoggedOut from "../components/LoggedOut.svelte";
+  import { auth } from "../stores/auth";
+  import { onMount } from "svelte";
+
+  onMount(() => {
+    window.global = window;
+    $auth.init();
+  });
+</script>
+
+<main>
+  {#if $auth.isReady}
```

Comment:
you can spare a if using `&&`

or use a derived store or local auto subscriber state that computes the value required